pkg/driver/controller.go

```diff
@@ -177,6 +186,13 @@ func (cs *controller) DeleteVolume(
 			)
 		}
 	}
+
+	sendEventOrIgnore(cvc.GetAnnotations()[utils.OpenebsPVC], volumeID, getCapacity(cvc),
+		strconv.Itoa(cvc.Spec.Provision.ReplicaCount),
+		"cstor-csi",
```

Comment:
(optional) You might want to use this as a constant(taken from the string constant and inside the sendEventOrIgnore method) from inside sendEventOrIgnore as this repository has a single applicationName/CasType.


This would be good, if you don't plan to include other repos as a dependency(as it'd become confusing).

Also, why did we move away from openebs/maya mono-ish repo?

Reply:
Its getting difficult to manage in mono repo , and to support latest development with csi etc , better idea to seperate them out .

Reply:
Yeah, opening mono-repos in vim is hard too :+1: 
Aside, I wonder how Google deals with 76TB of mono-repo, esp merge conflicts :laughing: